samples/async-context/worker.js

```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2022-2023 Cloudflare, Inc.
```

Comment:
Existing files seem to say 2017-2022, does that matter? https://github.com/cloudflare/workerd/blob/main/src/workerd/util/abortable.h#L1

(Ideally we'd omit the year numbers since updating them is a pain)

Reply:
The [guidance](https://wiki.cfdata.org/display/LAW/Open+Source+Compliance) includes a year range so this is just following that example.

The first year is important as the publication date (100 years copyright from then); the later year reflects when the file was last updated and people can update it if they care to, but no pressure.

The script uses some simple logic to determine the range:

* if the file is obviously part of the initial import go with 2017-2023 (as those files all cite 2017 as the publication date and 2023 is the current year for this modification)
* if the file added in a year prior to this year, go with <first_year>-<this_year>.
* if the file was added this year, go with <this_year>.

Small variations in the year range is very easy to handle if ever there needs to some copyright normalization (the boilerplate is otherwise identical).

Reply:
Good suggestion on CI, we could also have some new file templates that add the header with the year pre-populated.